charts/azuremonitor-containers-geneva/templates/deployment.yaml

```diff
@@ -61,12 +61,27 @@ spec:
            value: {{ .Values.genevaLogsConfig.region | quote }}
          - name: MONITORING_USE_GENEVA_CONFIG_SERVICE
            value: "true"
-         - name: MONITORING_MAX_EVENT_RATE
-           value: "100000" # default MDSD EPS is 20K which is not enough for large scale
          - name: FBIT_INPUT_FORWARD_BUFFER_CHUNK_SIZE
            value: "15m"
          - name: FBIT_INPUT_FORWARD_BUFFER_CHUNK_MAX_SIZE
            value: "30m"
+         # MDSD high scale config
+         - name: MONITORING_MAX_EVENT_RATE
+           value: "100000" # default MDSD EPS is 20K which is not enough for large scale
+         - name: MDSD_COMPRESSION_ALGORITHM
```

Comment:
Can we add the meaning of all these vars in our wiki or something?

Reply:
> Can we add the meaning of all these vars in our wiki or something?

These are all MDSD specific. let me check with MDSD team and see if they have doc for this.